styles/_header.scss

```diff
@@ -0,0 +1,89 @@
+.main-header {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+}
+.container-header {
+  padding-top: 50px;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+.nav-bar {
+  padding-top: 24px;
+  padding-bottom: 10px;
+}
+.toggle-lang {
+  display: inline-flex;
+}
+.toggle-lang__link {
+  display: inline-block;
+  padding: 5px 6px;
+  border-radius: 8px;
+  font-weight: 700;
+  font-size: 12px;
+  color: $grey-bg-text-color;
+  &:hover {
+    background-color: $accent-color;
+    color: white;
+    text-decoration: none;
+  }
+  @include media-breakpoint-up(xl) {
+    color: white;
+  }
+}
+.toggle-lang__link--active {
+  background-color: $accent-color;
+  color: white;
+  &:hover {
+    background-color: $accent-color;
+    color: white;
+    text-decoration: none;
```

Comment:
As I see you already have these styles for element class.

Reply:
Removed code that is not needed